.github/workflows/skipped-test-tracker-integration.yml

```diff
@@ -1,70 +0,0 @@
-name: Skipped Test Tracker Integration Test
```

Comment:
removed "unit test" workflows for skipped and flaky tests (they were duplicating the real workflows but just not uploading the data)